storageimg_kvm.py

```diff
@@ -0,0 +1,333 @@
+            required: true
+        storage:
+            description: The name of the storage image to be removed.
+            type: string
+            required: true
+    return:
+        description: |
+            A tuple with a boolean flag stating the remove was successful or not and
+            the output or error message.
+        type: tuple
+    """
+    # Define message
+    messages = {
+        1000: f'1000: Successfully removed storage image {storage} from {domain_path} on Host {host}.',
+        3021: f'3021: Failed to connect to the Host {host}',
+        3022: f'3022: Failed to remove storage image {domain_path}{storage} from Host {host}.'
+    }
+    message_list = {}
+    # Define payload
+    payload = f'rm --force {domain_path}{storage}'
+
+    # Remove storage using SSH communication
+    response = comms_ssh(
+        host_ip=host,
+        payload=payload,
+        username='robot',
+    )
+    if response['channel_code'] != CHANNEL_SUCCESS:
+        msg = f'{messages[3021]}\n ' \
+                f'channel_code: {response["channel_code"]}s.\n' \
+                f'channel_message: {response["channel_message"]}\n' \
+                f'channel_error: {response["channel_error"]}'
+        message_list.append(msg)
+        success = False
+    if response['payload_code'] != SUCCESS_CODE:
+        msg = f'{messages[3022]}\n ' \
+                f'payload_code: {response["payload_code"]}s.\n' \
+                f'payload_message: {response["payload_message"]}\n' \
+                f'payload_error: {response["payload_error"]}'
+        message_list.append(msg)
+        success = False
+
+    message_list.append(messages[1000])
+    return success, message_list
+
+
+def read(
+    host: str,
+    domain_path: str,
+    storage: str,
+):
+    """
+    description:
+        Gets the status of the <domain_path><storage> file info on the given Host <host>.
+
+    parameters:
+        host:
+            description: The dns or ipadddress of the Host where the storage image is located
+            type: string
+            required: true
+        domain_path:
+            description: The location or directory path where this storage image is read
+            type: string
+            required: true
+        storage:
+            description: The name of the storage image to be read
+            type: string
+            required: true
+    return:
+        description: |
+            A tuple with a boolean flag stating the read was successful or not and
+            the output or error message.
+        type: tuple
+    """
+    # Define message
+    messages = {
+        1000: f'1000: Successfully read storage image {storage}',
+        3021: f'3021: Failed to connect to the Host {host}',
+        3022: f'3022: Failed to read storage image {domain_path}{storage} on the Host {host}'
+    }
+    message_list = {}
+    data_dict = {
+        host: {
+            'image': None,
+            'disk_size': None,
```

Comment:
```suggestion
            'size': None,
```

Same here as mentioned in storage_kvm.read()